src/url-handling.js

```diff
@@ -90,10 +90,12 @@ export function urlStateToUrlObject(urlState: UrlState): UrlObject {
     range: stringifyRangeFilters(urlState.rangeFilters) || undefined,
     thread: urlState.selectedThread,
     threadOrder: urlState.threadOrder.join('-'),
-    hiddenThreads: urlState.hiddenThreads.join('-'),
     file: urlState.pathInZipFile || undefined,
     v: CURRENT_URL_VERSION,
   };
+  if (urlState.hiddenThreads.length > 0) {
```

Comment:
Could you please just add an empty line and a comment before the `if` condition, explaining something like `Add the parameter hiddenThreads only when needed` ?